src/main.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+import faker from 'faker';
+import cowsay from 'cowsay-browser';
+import './style/main.scss';
+
+class Header extends React.Component {
+  render() {
+    return (<header>
+      <h1>Generate Cowsay Lorem</h1>
+    </header>
+    );
+  }
+}
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      content: cowsay.say({ text: `${faker.lorem.words(3)}` }),
+    };
+    this.handleCowsayMessage = this.handleCowsayMessage.bind(this);
+  }
+
+  handleCowsayMessage() {
+    this.setState(() => {
+      return {
+        content: cowsay.say({ text: `${faker.lorem.words(3)}` }),
+      };
+    });
+  }
+
+  render() {
+    return (
+      <div>
+        <Header/>
+        <button onClick = { this.handleCowsayMessage }> click me </button>
```

Comment:
I won't knock you down for this spacing at Line 37 since it doesn't really break anything. Just want to emphasize if you're going to stick with this style of extra spacing, be consistent with it in future code.